### PR TITLE
Updates to improve use by newcomers

### PR DESCRIPTION
**Pull request by sminnee**
* Use the standard scaffolder
 * Add a readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,61 @@
...
+    uri: 'http://localhost:8080/graphql/',
+    opts: {
+        credentials: 'same-origin',
+    },
+});
+```
+
+ * If you haven't yet installed Ionic, install that with `npm install -g cordova ionic`.
+ * On first download, run `npm install` in the root of your project folder.
+ * Start Ionic with `ionic serve`.
```
Comment (Firesphere) [inline README.md]: You can also start the project in an emulator, as per the documentation:
http://ionicframework.com/docs/v1/guide/testing.html

`ionic cordova run Android`

Make sure you either have an Android device connected, or have a virtual Android device running.

**Comment by Firesphere** [comment]
Could you squash it into a single commit when it's ready to go?

**Comment by sminnee** [comment]
OK changed docs and added a loading indicator...

File: src/pages/staff/staff.ts [hunk 2]
```diff
@@ -32,10 +33,13 @@ export class StaffPage implements OnInit {
...
-            query: MemberList
-        });
-        this.loading = false;
+      const self = this;
+      this.apollo.watchQuery<QueryResponse>({
+          query: MemberList
+      }).subscribe(function (result) {
+          self.staffMembers = (result.data as any).readMembers.edges.map(edge => edge.node);
+          self.loading = false;
+      });
```
Comment (Firesphere) [inline src/pages/staff/staff.ts:42]: 👍

File: README.md [hunk 3]
```diff
@@ -1,5 +1,62 @@
...
+        SilverStripe\Security\Member:
+          fields: [ID, Name, FirstName, Surname, Email]
+          operations:
+            create: true
+            read: true
+```
+
+Then install the JWT authentication package:
+
+    composer require firesphere/silverstripe-graphql-jwt:^1.0@dev
```
Comment (Firesphere) [inline README.md:40]: Could be wrapped inside backticks probably?

File: README.md [hunk 4]
```diff
@@ -1,5 +1,62 @@
...
+const networkInterface = createNetworkInterface({
+    uri: 'http://localhost:8080/graphql/',
+    opts: {
+        credentials: 'same-origin',
+    },
+});
+```
+
+ * If you haven't yet installed Ionic, install that with `npm install -g cordova ionic`.
+ * On first download, run `npm install` in the root of your project folder.
```
Comment (Firesphere) [inline README.md:57]: Note to self: Make it Yarn because it's still faster.

File: README.md (hunk 3 above)
Comment (sminnee) [reply README.md:40]: The indent does that.

File: src/pages/staff/staff.html [hunk 5]
```diff
@@ -11,12 +11,13 @@
...
     </ion-refresher>
-    <h2>Hello Members!</h2>
+    <h2 *ngIf="!loading">Hello Members!</h2>
+    <h2 *ngIf="loading">loading...</h2>
     <ion-item>
-        <ul>
-            <li *ngFor="let entry of data | async | select: 'readMembers'">
-                Name: {{entry.Name}} {{entry.Surname}}<br/>
-                Email: {{entry.Email}}
+        <ul *ngIf="!loading">
```
Comment (Firesphere) [inline src/pages/staff/staff.html:17]: Loving the loading addition!

File: src/pages/staff/staff.ts [hunk 6]
```diff
@@ -5,11 +5,12 @@ import gql from 'graphql-tag';
 
 // We use the gql tag to parse our query string into a query document
 const MemberList = gql`
-  query { readMembers {
-    Email: Email,
-    Name: FirstName,
-    Surname: Surname
-}}`;
+  query { readMembers { edges { node {
+    Email
```
Comment (Firesphere) [inline src/pages/staff/staff.ts:9]: You removed the mapping, understandable, but I believe leaving it in would be a good addition to demo the power of GraphQL?

File: README.md (hunk 3 above)
Comment (Firesphere) [reply README.md:40]: Okido, cool :) GitHub Markdown === magic to me sometimes

**Comment by Firesphere** [comment]
If you could put back the mapping, I'm okay with this and loving it :)
https://github.com/Firesphere/silverstripe-headless-ionic/pull/1#discussion_r128910523

Excellent documentation. Thanks!

**Comment by Firesphere** [comment]
looks awesome and works on my machine. Assuming it does on yours too. Merging :)

**Pull request merged**